core/sessions/orm.go

```diff
@@ -172,10 +176,12 @@ func (o *orm) CreateSession(sr SessionRequest) (string, error) {
 	// Do email and password check first to prevent extra database look up
 	// for MFA tokens leaking if an account has MFA tokens or not.
 	if !constantTimeEmailCompare(strings.ToLower(sr.Email), strings.ToLower(user.Email)) {
+		o.auditLogger.Audit(context.Background(), audit.AuthLoginFailedEmail, map[string]interface{}{"email": sr.Email})
```

Comment:
It's probably no longer necessary to pass a context in here since it makes no external requests now.